REACT/tekgain/src/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
```

Comment:
Remove reportWebVitals.js and it's corresponding import from index.js  and remove all files from Public folder except index.html